Documentation/concepts/networking/ipam/eni.rst

```diff
@@ -450,3 +450,42 @@ Metrics
 *******
 
 The IPAM metrics are documented in the section :ref:`ipam_metrics`.
+
+******************
+Node Configuration
+******************
+
+The IP address and routes on ENIs attached to the instance will be
+managed by the Cilium agent. Therefore, any system service trying to manage
+newly attached network interfaces will interfere with Cilium's configuration.
+Common scenarios are ``NetworkManager`` or ``systemd-networkd`` automatically
+performing DHCP on these interfaces or removing Cilium's IP address when the
+carrier is temporarily lost. Be sure to disable these services or configure
+your Linux distribution to not manage the newly attached ENI devices.
+The following examples configure all Linux network devices named ``eth*``
+except ``eth0`` as unmanaged.
+
+.. tabs::
+
+   .. group-tab:: Network Manager
```

Comment:
For future reference these tabs don't need to be [grouped](https://devopstutodoc.readthedocs.io/en/latest/doc_generators/sphinx/contributed_extensions/sphinx-tabs/sphinx-tabs.html#grouped-tabs), I think [basic tabs](https://devopstutodoc.readthedocs.io/en/latest/doc_generators/sphinx/contributed_extensions/sphinx-tabs/sphinx-tabs.html#basic-tabs) would work too. But not an issue and it's not as if we paid a lot of attention to it in the rest of the documentation anyway.

Reply:
I actually didn't know that basic tabs existed, thanks for pointing this out! If we ever add an additional `tabs` section to expand on the required network config on this page, they most likely would be grouped together with the above. So I'll leave them as they are for now.